src/gluonts/model/deepvar/_network.py

```diff
@@ -860,3 +878,19 @@ def hybrid_forward(
             past_is_pad=past_is_pad,
             future_time_feat=future_time_feat,
         )
+
+    def post_process_samples(self, samples: Tensor) -> Tensor:
+        """
+        Identity map.
```

Comment:
Maybe you can describe what this method does in general, ie how it is used and how it affects the model, so that it’s clear what one can achieve by overriding it.

Reply:
Done.